apps/api/package.json

```diff
@@ -119,11 +119,11 @@
     "typescript": "4.9.5"
   },
   "optionalDependencies": {
-    "@novu/ee-auth": "^0.24.2",
-    "@novu/ee-billing": "^0.24.2",
-    "@novu/ee-echo-api": "^0.24.2",
-    "@novu/ee-shared-services": "^0.24.2",
-    "@novu/ee-translation": "^0.24.2"
+    "@novu/ee-auth": "workspace:",
```

Comment:
@rifont This is gold! It was one of the big blockers to bundling all our source code in a single file in Nest.